### PR TITLE
Add caption cmsvieira1970

### PR DESCRIPTION
**Pull request by cmsvieira1970**
@nunomichellsousa  Issue

### CODE REVIEW

File: cv2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+ZELLO
```
Comment (cmsvieira1970) [inline cv2.txt:1]: teste

**Review by cmsvieira1970 — COMMENTED** [review]
ok

File: _slides/12-cmsvieira1970.md [hunk 2]
```diff
@@ -3,7 +3,10 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Git Hub is the Best
```
Comment (cmsvieira1970) [inline _slides/12-cmsvieira1970.md:6]: ok

File: _slides/12-cmsvieira1970.md [hunk 3]
```diff
@@ -3,7 +3,10 @@ layout: slide
...
+
+Git Hub is the best of all
```
Comment (cmsvieira1970) [inline _slides/12-cmsvieira1970.md:8]: ok

File: _slides/12-cmsvieira1970.md (hunk 2 above)
Comment (cmsvieira1970) [reply _slides/12-cmsvieira1970.md:6]: ok

**Review by cmsvieira1970 — COMMENTED** [review]
ok

**Review by nicruo — APPROVED** [review]

**Pull request merged**